kubernetes/k8s_workflow_integration.go

```diff
@@ -37,7 +36,7 @@ import (
 
 // ServerlessWorkflowSpec defines a base API for integration test with operator-sdk
 type ServerlessWorkflowSpec struct {
-	model.Workflow `json:",inline"`
+	// model.Workflow `json:",inline"`
```

Comment:
Leftovers?

Reply:
It was kept to adapt to the Kubernetes Controller in the future.
Will this Kubernetes folder be kept?

Reply:
Yes, the SDK must be compatible with Kubernetes objects.

Reply:
The kubernetes controller-gen requires `struct` to work.

I tested the tool (https://github.com/omissis/go-jsonschema) to generate the `struct`, but it gives an error with `interface` when generating the code with deepcopy.

Using go-jsonschema or another tool seems like a solution.